tests/www/test_utils.py

```diff
@@ -190,6 +191,43 @@ def some_func():
                 self.assertEqual(anonymous_username, kwargs['owner'])
                 mocked_session_instance.add.assert_called_once()
 
+    def test_get_python_source_from_method(self):
+        class AMockClass(object):
+            def a_method(self):
+                """ A method """
+                pass
+
+        mocked_class = AMockClass()
+        print(utils.__file__)
```

Comment:
Probably shouldn't leave this in here :)

Reply:
oops!